dKoP_oM/IdKopObject.cs

```diff
@@ -0,0 +1,11 @@
+﻿using BH.oM.Base;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BH.oM.dKoP
+{
+    public interface IdKopObject : IObject
```

Comment:
```suggestion
    public interface IdKoPObject : IObject
```

To align with convention elsewhere